packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -78,19 +78,21 @@ function CardCategory({
     return null;
   }
 
+  console.log(item);
+
   return (
     <CardLayout
       href={href}
       icon="🗃️"
       title={item.label}
       description={translate(
         {
-          message: '{count} items',
+          message: '({count} items) {description}',
```

Comment:
This is up for discussion. I personally prefer the `description` to override the default "x items" description, since we only have one precious line of space (which I think should be extended)

Reply:
@Josh-Cena Alright! I proposed this so because I didn't know what was idea to deal with description, but in any case if you want any change I'll do 😊  hope to finish it for the hacktoberFest 😂

Reply:
agree this label looks weird. And it makes tests fails

> hope to finish it for the hacktoberFest 😂 

I only work on Docusaurus starting Wednesday so this should be merged before Friday then 🤪